lib/std/crypto/25519/edwards25519.zig

```diff
@@ -75,15 +75,6 @@ pub const Edwards25519 = struct {
         .is_base = true,
     };
 
-    /// The edwards25519 neutral element.
-    pub const neutralElement = Edwards25519{
```

Comment:
I know nothing about this, but it is marked `pub`. Just double checking, are std lib users expecting to find this constant available?

Reply:
It is equivalent to `identityElement`.

But since `identityElement` is private, let's make that one public, and add a deprecation compile error log for `neutralElement`.